lib/OpenQA/Controller/API/V1/Iso.pm

```diff
@@ -38,11 +38,19 @@ sub _sort_dep {
         $added = 0;
         for my $job (@$list) {
             next if $done{$job};
-            my $after = $job->{START_AFTER_TEST};
-            if (!defined $after || $count{$after} == 0) {
+            my @after;
+            push @after, split(/\s*,\s*/, $job->{START_AFTER_TEST}) if defined $job->{START_AFTER_TEST};
+            push @after, split(/\s*,\s*/, $job->{PARALLEL_WITH}) if defined $job->{PARALLEL_WITH};
+
+            my $c = 0;
+            foreach my $a (@after) {
```

Comment:
"The foreach keyword is actually a synonym for the for keyword" - and as you use for later, it would be useful to have this consistent